index.html

```diff
@@ -71,1285 +81,1323 @@
+      </p>
+      <p>
+        Value by default: <code>false</code>.
+      </p>
+      <p>
+        To <dfn>process the window's <code>auto_design_width</code> member</dfn>, given [=object=] |json:JSON| and [=ordered map=] |window:ordered map|:
+      </p>
+      <ol class="algorithm">
+        <li>If |json|["auto_design_width"] does not [=map/exist=], or if the type of |json|["auto_design_width"] is not [=boolean=], return.</li>
+        <li>Set |window|["auto_design_width"] to |json|["auto_design_width"].</li>
+      </ol>       
+    </section>
+    <section data-cite="CSS-COLOR-4 CSS-SYNTAX-3 APPMANIFEST">
+      <h3><code>background_color</code> member</h3>
+      <p>
+        The <dfn data-dfn-for="MiniApp window resource"><code>background_color</code></dfn> member is a [=string=] that specifies the background color of the window that contains a MiniApp.
+      </p>
+      <p>
+        This member supports [=sRGB=] colors, and it is equivalent to the application manifest's <a href="https://www.w3.org/TR/appmanifest/#dfn-background_color" data-link-type="dfn"><code>background_color</code></a>.
+      </p>
+      <p>
+        Value by default: <code>"#ffffff"</code>.
+      </p>
+      <p class="note" title="Processing the `background_color` member">
+        When [=processing a MiniApp manifest=], the <a href="https://w3c.github.io/manifest/#dfn-process-a-color-member" data-link-type="dfn">process a color member</a> algorithm is used to process the window's <code>background_color</code> member, according to the [[APPMANIFEST]] specification.
+      </p>      
+    </section>
+    <section data-cite="CSS-COLOR-4 MEDIAQUERIES">
       <h3><code>background_text_style</code> member</h3>
       <p>
-        <span its-locale-filter-list="en" lang="en">The <dfn>background_text_style</dfn> member specifies the background text style.</span>
-        <span its-locale-filter-list="zh-hans" lang="zh-hans">背景文字样式，。</span>
+        <span>The <dfn><code>background_text_style</code></dfn> member is a [=string=] that specifies the background text style, indicating a light or dark color theme.</span>
       </p>
-      <p class="note">
-        <span its-locale-filter-list="en" lang="en">
-          Value by default: <code>dark</code>.
-        </span>
-        <span its-locale-filter-list="zh-hans" lang="zh-hans" class="translateme">
-          Value by default: <code>dark</code>.
-        </span>
+      <p>
+        This member supports <a data-xref-type="css-descriptor" data-xref-for="@media">prefers-color-scheme</a> and MUST contain one of the following <dfn>background text style values</dfn>:  
```

Comment:
```suggestion
        This member supports the values of <a data-xref-type="css-descriptor" data-xref-for="@media">prefers-color-scheme</a> and MUST contain one of the following <dfn>background text style values</dfn>:  
```

According to my understanding, we do not directly support the CSS `prefers-color-scheme` media feature, but only support the values of it.